src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -977,6 +978,12 @@ private function registerSerializerConfiguration(array $config, ContainerBuilder
             $definition->addTag('serializer.normalizer', array('priority' => -900));
         }
 
+        if (class_exists(YamlEncoder::class)) {
```

Comment:
The existing check is useful though, to keep compatibility with serializer 3.1 (other checks above also ensure compat with different versions of the component)